client/index.js

```diff
@@ -17,6 +20,19 @@ import {
   saveToColdStorage
 } from './cold-reload';
 
+const localhostRE = /^(localhost|127.|0.)/;
```

Comment:
Instead of filtering out in localhost we should just disable when NODE_ENV === 'development'. This makes it easier to test and falls in line with our serverside opbeat setup